test/ShortestPaths/runtests.jl

```diff
@@ -0,0 +1,37 @@
+using SparseArrays
+using LightGraphs.ShortestPaths
+using LightGraphs.Traversals
+using LightGraphs.Traversals: NOOPSort
+
+import Base.==
+function ==(a::ShortestPaths.AStarResult, b::ShortestPaths.AStarResult)
+   return a.path == b.path && a.dist == b.dist
+end
+
+const sptestdir = dirname(@__FILE__)
+
+sptests = [
+    "astar.jl",
+    "bellman-ford.jl",
+    "bfs.jl",
+    "desopo-pape.jl",
+    "dijkstra.jl",
+    "distributed-dijkstra.jl",
+    "floyd-warshall.jl",
+    "johnson.jl",
+    "distributed-bellman-ford.jl",
+    "distributed-johnson.jl",
+    "spfa.jl",
+    "threaded-bfs.jl",
+    "threaded-floyd-warshall.jl"
+]
+
+@testset "LightGraphs.ShortestPaths" begin
+    @test !ShortestPaths.has_negative_weight_cycle(path_graph(5))
+    @test !ShortestPaths.has_negative_weight_cycle(path_digraph(5))
```

Comment:
Why do we these tests here?

Reply:
Why wouldn't we? They're the analog to `has_negative_edge_cycle` in the old shortest paths code.

Reply:
I just thought, it is a bit weird, that these tests are in runtests.jl and not in their own file

Reply:
Oh. because the methods being tested are in `src/ShortestPaths/ShortestPaths.jl`:
```
has_negative_weight_cycle(g::AbstractGraph) = false
```